design-system/uilib/src/components/dpdhl-segmented-control/dpdhl-segmented-control.stories.svelte

```diff
@@ -0,0 +1,31 @@
+
+<script>
+    import { Meta, Template, Story } from '@storybook/addon-svelte-csf'
+    import { action } from '@storybook/addon-actions'
+    import '@sprinteins/dpdhl-uilib'
+
+</script>
+
+
+<Meta title="11_Components/Segmented Control" component={null}/>
+
+<Template let:args>
+    <div class="container">
+        <dpdhl-segmented-control items={args.items}/>
```

Comment:
using complex structures (objects) in attributes makes it hard to use the component in pure HTML.

Would it be possible to use the children pattern?
something along the lines of:

```svelte
<dpdhl-segmented-control>
    <dpdhl-segmented-control-item label="Menu 1" on:click={() => {}}>
    <dpdhl-segmented-control-item label="Menu 2" on:click={() => {}}>
    <dpdhl-segmented-control-item label="Menu 3" on:click={() => {}}>
</dpdhl-segmented-control>

```